app/src/main/java/com/github/se/eventradar/ui/navigation/NavigationActions.kt

```diff
@@ -15,6 +15,7 @@ object Route {
 
   // Event radar secondary screens
   const val EVENT_DETAILS = "event_details"
+  const val EVENT_DETAILS_TICKETS = "event_details_tickets"
```

Comment:
I'm not sure I understand what would be the use to have these in the string file ? For the TopLevelDestination it makes sense since their name is displayed on the bar uppon selection, but for smaller routes I don't know